py3status/modules/spotify.py

```diff
@@ -91,9 +95,10 @@ class Py3status:
         "stereo",
         "version",
     ]
+    dbus_client = "org.mpris.MediaPlayer2.spotify"
 
     def _spotify_cmd(self, action):
-        return SPOTIFY_CMD.format(cmd=action)
+        return SPOTIFY_CMD.format(dbus_client=self.dbus_client,cmd=action)
```

Comment:
python lint will require a space after the comma please

Reply:
Oh wow, that's a wrong push, sorry for that.